### PR TITLE
Add content to about me page

### PR DESCRIPTION
**Pull request by hananeikhelef**
-  Added content to the about me page 

- Added links to my linkedin and github accounts (as listed in the walkthroughs)

Demo : https://hananeikhelef-step-2020.uc.r.appspot.com/

TO DO's : 

- Add animation to the portfolio and fix some CSS 
- Add a contact page cause i'll need it for the server part.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,34 +1,63 @@
...
+               </ul>
+            </div>
+         </div>
+      </div>
+      <div id="content">
+         <section>
+            <h2>About me </h2>
+            <p> Hey !  I'm Hanane a software enginnering student at Polytechnique Montreal.
+               Super passionate about software development.
+               I previously interned at Busbud and now i'm intrening at Google.
```
Comment (mctavish) [inline portfolio/src/main/webapp/index.html:35]: Typos: interned, interning

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,34 +1,63 @@
...
+         <p>
+            On the Tech side, i worked more on web development using different frameworks and technologies. 
+         </p>
+         <section>
+            <h2>Hobbies</h2>
+            <p>
+               I enjoy volunteering whenever i have extra time especially in the tech field: 
+            <ul>
+               <li> I volunteered with AIESEC in egypt to teach Mathematics and french  </li>
+               <li> I am a memeber at the Women Tech Makers organization in Montreal </li>
```
Comment (mctavish) [inline portfolio/src/main/webapp/index.html:49]: Typo: member

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,34 +1,63 @@
...
+         <h2>Skills</h2>
+         <p>
+            On the Tech side, i worked more on web development using different frameworks and technologies. 
+         </p>
+         <section>
+            <h2>Hobbies</h2>
+            <p>
+               I enjoy volunteering whenever i have extra time especially in the tech field: 
+            <ul>
+               <li> I volunteered with AIESEC in egypt to teach Mathematics and french  </li>
```
Comment (mctavish) [inline portfolio/src/main/webapp/index.html:48]: It's worthwhile to add links to other sites when it makes sense.  Many of these would have websites that you could point readers to.

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -1,34 +1,63 @@
...
+         <section>
+         <h2>Skills</h2>
+         <p>
+            On the Tech side, i worked more on web development using different frameworks and technologies. 
+         </p>
+         <section>
+            <h2>Hobbies</h2>
+            <p>
+               I enjoy volunteering whenever i have extra time especially in the tech field: 
+            <ul>
```
Comment (mctavish) [inline portfolio/src/main/webapp/index.html:47]: Unfortunately, the bullets along with the centred text don't work well together.  I don't have a concrete suggestion for how to fix it, unfortunately.

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -1,34 +1,63 @@
...
+               <li> I organize and plan hackathons in my school </li>
+            </ul>
+            </p>
+         </section>
+      </div>
+      <footer>
+         <p> Hanane ikhelef<br>
+         <a href="mailto:hege@example.com">hananeikhelef@gmail.com</a>
+         </p>
+         <a href="https://www.linkedin.com/in/hanane-ikhelef-970714168/?trk=public_profile_samename_profile_profile-result-card_result-card_full-click&originalSubdomain=ca"><i class="fa fa-linkedin"></i></a>
```
Comment (mctavish) [inline portfolio/src/main/webapp/index.html:59]: Nit: The linkedin url looks like it has more baggage than is necessary.  Unless you want the link tracking component for some reason, I'd suggest that you remove it.

File: portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -1,34 +1,63 @@
...
-           <img src="/images/hanane.jpeg" alt="Trulli" >
-       </div>
-
-    <footer>
-     <p> Hanane ikhelef<br>
-     <a href="mailto:hananeikhelef@gmail.com">hananeikhelef@gmail.com</a></p>
-    </footer>
-
-  </body>
+   <head>
```
Comment (mctavish) [inline portfolio/src/main/webapp/index.html:3]: I noticed that the css doesn't play well with small window resolutions.  You don't need to fix it in this PR, but it's something that you should look into going forward.

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (hananeikhelef) [reply portfolio/src/main/webapp/index.html:47]: I checked on stackoverflow, it says to add  **list-style-position: inside** to the **ul** element.
Here's the link : https://stackoverflow.com/questions/28977320/how-do-i-get-the-bullet-points-of-a-ul-to-center-with-the-text

**Pull request merged**